### DISCUSSION TITLE
Is it possible to add support for the ILI6485A chip?

### DISCUSSION THREAD

**Discussion by JKeyRandom** [Bodmer/TFT_eSPI#2481] [New processor/display support or capability request] [answered]
I have recently seen a manufacturer that offers a screen with ESP32 incorporated and tells me that it has the ILI6485A chip and the GT911 touch, I know that it is compatible with the Arduino GFX library, I would like to use the TFT_eSPI library together with GUIslice as I have been doing so far with ILI9341 screens.

[Link screen](https://es.aliexpress.com/item/1005004974870929.html?spm=a2g0o.store_pc_allProduct.8148356.19.2f067a42DMorUy&pdp_npi=2%40dis%21EUR%21%E2%82%AC%2040%2C51%21%E2%82%AC%2024%2C31%21%21%21%21%21%40211b88ef16783005300987670efa5f%2112000031242398706%21sh)

[Arduino GFX Library](https://github.com/moononournation/Arduino_GFX)

**Comment by Bodmer** [comment, marked as answer]
The RGB interface and that controller are not supported and are unlikely to be supported in future.

**Reply by JKeyRandom** [reply]
Thank you for your response. I take this opportunity to congratulate you on your great work with this library.
